reflex/components/component.py

```diff
@@ -446,25 +446,24 @@ def __str__(self) -> str:
 
         return _compile_component(self)
 
-    def _apply_theme(self, theme: Component):
+    def _apply_theme(self, theme: Optional[Component]):
```

Comment:
I think this should not be Optional, or else we will have to do `None` checks within every component's `_apply_theme` method.